lib/engine/share.rb

```diff
@@ -29,7 +29,11 @@ def id
     end
 
     def num_shares
-      @percent / corporation.share_percent
+      num_shares_float.ceil
```

Comment:
can we do the inverse? keep num_shares the raw value

@percent.to_f / corporation.share_percent

Reply:
you could do def num_shares(ceil: true / false)

num = @percent.to_f / corporation.share_percent
ceil ? num.ceil : num

Reply:
Ok, updated